bower.json

```diff
@@ -17,7 +17,8 @@
     "tests"
   ],
   "devDependencies": {
-    "jquery": "^2.2.3"
+    "jquery": "^2.2.3",
+    "flexslider": "^2.6.1"
```

Comment:
This is a jQuery plugin that exists in the clickhole site. We may want it explicitly in this code but for now it's just something I need in the tests.